### DISCUSSION TITLE
Torodd F. Ottestad - Jordskjelvpython

### DISCUSSION THREAD

**Discussion by giscus** [lektorodd/lektorodd.github.io#5] [General]
# Torodd F. Ottestad - Jordskjelvpython

https://lektorodd.github.io/posts/230425_jordskjelv/

**Comment by MBult-81** [comment]
Hei Torodd,
Jeg er ny i Anaconda og prøvde å følge ditt råd om å lage et nytt environment som følgende:
conda create -n geo_env
conda activate geo_env
conda config --env --add channels conda-forge
conda config --env --set channel_priority strict
conda install python=3 geopandas
Jeg klarer med dette å skape geo_env men det inneholder ikke Jupyter apps. Kan jeg lett få tilgang til Jupyter der?

Ellers bruker jeg:
pip install geopandas
i base(root) environment og det virker greit.

Takk for koden :-)

Mvh
Matthieu

**Reply by lektorodd** [reply]
Min feil! 

Jupyter Lab må òg installerast. Så viss du no skriv
```{shell}
conda install jupyterlab
```
når du er i `geo_env` så skal det funka 😄 

Skal oppdatera oppskrifta i blogginnlegget til
```{shell}
conda create -n geo_env
conda activate geo_env
conda config --env --add channels conda-forge
conda config --env --set channel_priority strict
conda install python=3 geopandas jupyterlab 
```

Så ved oppstart av Jupyter Lab i `geo_env` bruker eg 
```{shell}
conda activate geo_env 
jupyter lab
```
